index.es6.js

```diff
@@ -87,7 +88,13 @@ export default React.createClass({
           onBlur={this.handleBlur}
           className="react-toggle-screenreader-only"
           type="checkbox"
-          {...this.props} />
+          onChange={this.props.onChange}
+          name={this.props.name}
+          value={this.props.value}
+          id={this.props.id}
+          disabled={this.props.disabled}
+          aria-labelledby={this.props['aria-labelledby']}
+          aria-label={this.props['aria-label']} />
```

Comment:
Instead of explicitly writing this out, you can move `{...this.props}` up above `className="react-toggle-screenreader-only"`, that way, the `className="..."` will overwrite the one from `{...this.props}` :)